ocrd_page_to_alto/convert.py

```diff
@@ -179,11 +179,10 @@ def convert_reading_order(self):
             self.alto_printspace.find('.//*[@ID="%s"]' % id_cur).set('IDNEXT', id_next)
 
     def convert_border(self):
-        page_width = self.page_page.imageHeight
+        page_width = self.page_page.imageWidth
```

Comment:
Ouch 😬 Thanks for spotting.

Reply:
That's particularly relevant because Kitodo.Presentation / DFG-Viewer uses `@WIDTH` alone for coordinate calculations.  It adds a footer and to that end needs to scale the image up to multiples of 1000 or 2000 px; all ALTO coordinates are then scaled accordingly:

https://github.com/kitodo/kitodo-presentation/blob/9113a5e647b9142dde98e09a03a18d53d99bb077/Resources/Public/Javascript/PageView/AltoParser.js#L218-L244